concepts/index.md

```diff
@@ -19,11 +19,23 @@ You use Spinnaker's cluster management features to view and manage your resource
 
 ### Server Group
 
-The base resource, the *Server Group*, identifies the machine instance profile on which to execute images along with the number of instances. This resource is associated with a Load Balancer and a Security Group. A Server Group also has basic configuration settings, such as user account information and the region/zone in which images are deployed. When deployed, a Server Group is a collection of virtual machines running software.
+The base resource, the *Server Group*, identifies the machine instance profile on which to 
```

Comment:
@skim1420, what is a "machine instance profile"? Also "base resource" relative to what? This wording is fuzzy & seems to use terms defined nowhere else.

How about:
A *Server Group* is a Spinnaker abstraction for a set of *Instances*, each configured identically, and running the same exact *Artifact*. This abstraction maps to different resources in different cloud providers. For example, in the Google Compute Engine provider, a *Server Group* is a [Managed Instance Group](https://cloud.google.com/compute/docs/instance-groups/#managed_instance_groups). In the Amazon Web Services provider it maps to an [Auto Scaling Group](http://docs.aws.amazon.com/autoscaling/latest/userguide/AutoScalingGroup.html).